Updated paragraph on Reporting Security Vulnerabilities (#534)
mulesoft/docs-runtime-manager: modules/ROOT/pages/penetration-testing-policies.adoc

** Steps to reproduce the vulnerability
** All applicable HTTP requests and responses
** Explanation of why the example is considered a finding
--

=== Reporting Security Vulnerabilities

> To report security vulnerability findings, please follow the https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide]. Security vulnerability reports that do not appropriately follow the required steps will be rejected.

Reviewer: ```suggestion
To report security vulnerability findings, follow the https://help.salesforce.com/articleView?id=000320207&type=1&mode=1[Security Vulnerability Finding Submittal Guide]. Security vulnerability reports that do not follow the required steps will be rejected.
```

> Please, note that Salesforce does not address any security vulnerabilities that fall under the following categories:

Reviewer: ```suggestion
Salesforce does not address any security vulnerabilities in the following categories:
```

* Scan output from automated vulnerability scanning tools without any valid proof of concept

> * Security bug without a valid proof of concept

Reviewer: ```suggestion
* Security bugs without valid proof of concept
```

> * Vulnerabilities identified in an old or deprecated version of Mule runtime

Reviewer: ```suggestion
* Vulnerabilities identified in old or deprecated versions of Mule runtime engine
```

> * Vulnerability induced via code customization made by your company

Reviewer: ```suggestion
* Vulnerabilities introduced via code customization made by your company
```
